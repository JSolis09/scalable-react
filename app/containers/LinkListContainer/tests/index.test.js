// import React from 'react';
// import { shallow } from 'enzyme';

import { LinkListContainer } from '../index';

describe('<LinkListContainer />', () => {
  it('should render correctly', () => {
    expect(LinkListContainer).toBeDefined();
    // const linkListContainer = shallow(<LinkListContainer />);
    // expect(linkListContainer).toMatchSnapshot();
  });
});
