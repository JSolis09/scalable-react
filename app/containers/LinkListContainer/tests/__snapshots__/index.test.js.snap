// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkListContainer /> should render correctly 1`] = `<LinkList />`;
